spec/Section 6 -- Execution.md

```diff
@@ -236,7 +236,7 @@ connectivity.
 **Delivery Agnostic**
 
 GraphQL subscriptions do not require any specific serialization format or
-transport mechanism. Subscriptions specifies algorithms for the creation of a
+transport mechanism. Subscription specifies algorithms for the creation of a
```

Comment:
“The Subscription operation specifies…”

Reply:
#959 preferred

Reply:
So are we telling @rivantsov to revert this grammar change in favor of @benjie pr...?

Reply:
guys, it is left untouched (I reverted my change) in favor of benji's, except this grammatic fix, plural to singular.

Reply:
Above, I asked for clarification from the prior reviewers whether they would advise merging your line fix here or abandoning it. They responded that you should abandon it in favor of a different change. My later comment was meant to clarify this for you so the remainder of the edits can be merged.

My interpretation as to why your grammatical fix here should be abandoned entirely rather than accepted as an incremental improvement is because although it does correct one error (single vs plural), it introduces an orthogonal one.

But my interpretation here doesn't matter. I am just trying to better convey the prior reviewers feedback so that this can be merged as fast as possible.